content/en/blog/_posts/2018-12-04-kubeadm-ga-release.md

```diff
@@ -23,7 +23,7 @@ To keep kubeadm lean, focused, and vendor/infrastructure agnostic, the following
 - Non-critical add-ons, e.g. for monitoring, logging, and visualization
 - Specific cloud provider integrations
 
-Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the [Cluster API](https://github.com/kubernetes-sigs/cluster-api). Instead, kubeadm covers only the common denominator in every Kubernetes cluster: the [control plane](/docs/concepts/#kubernetes-control-plane). The user may install their preferred networking solution and other add-ons on top of Kubernetes *after* cluster creation.
+Infrastructure provisioning, for example, is left to other SIG Cluster Lifecycle projects, such as the [Cluster API](https://github.com/kubernetes-sigs/cluster-api). Instead, kubeadm covers only the common denominator in every Kubernetes cluster: the [control plane](/docs/reference/glossary/?all=true#term-control-plane). The user may install their preferred networking solution and other add-ons on top of Kubernetes *after* cluster creation.
```

Comment:
What do you think about linking the control plane to:
/docs/concepts/architecture/control-plane-node-communication/

Reply:
Perhaps using a glossary tooltip as what @sftim said would be a better choice, right? We could add a page to introduce control plane in detail in the future.